src/locales/en/modals/translation.json

```
@@ -72,5 +72,11 @@
 		"readme": "README",
 		"nameRequired": "Name is required",
 		"nameTaken": "Name is already taken"
+	},
+	"warningActiveDeployment": {
+		"title": "Warning",
+		"warning": "Changes might affect the currently running deployments",
```

Like in the other modals, the warning content should be content.

```suggestion
		"content": "Changes might affect the currently running deployments",
```

---

changed